reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -37,9 +43,24 @@
 final class MicrometerHttpServerMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpServerMetricsRecorder {
 
 	final static MicrometerHttpServerMetricsRecorder INSTANCE = new MicrometerHttpServerMetricsRecorder();
+	private final static String HTTP = "http";
+
+	protected final Counter.Builder activeConnectionsBuilder;
+	protected final ConcurrentMap<MeterKey, Counter> activeConnectionsCache = new ConcurrentHashMap<>();
+
+	protected final Counter.Builder connectionsBuilder;
+	protected final ConcurrentMap<MeterKey, Counter> connectionsCache = new ConcurrentHashMap<>();
 
 	private MicrometerHttpServerMetricsRecorder() {
-		super(HTTP_SERVER_PREFIX, "http");
+		super(HTTP_SERVER_PREFIX, HTTP);
+
+		this.activeConnectionsBuilder =
+				Counter.builder(reactor.netty.Metrics.HTTP_SERVER_PREFIX + ACTIVE_CONNECTIONS)
+						.description("The number of http connections currently processing requests");
+
+		this.connectionsBuilder =
+				Counter.builder(HTTP_SERVER_PREFIX + TOTAL_CONNECTIONS)
+						.description("The number of all opened connections");
```

Comment:
this one is not needed. `super(HTTP_SERVER_PREFIX, HTTP)` creates a counter builder with http both in the name and as a `URI` tag. actually, I don't think you need to override `incrementServerConnections` thanks to that fact

```suggestion
```